src/components/Logo/Logo.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import logo from './logo.svg';
+import './Logo.css';
+import {NavLink} from 'react-router-dom';
```

Comment:
I usually pad these with a space, makes it look a bit nicer

``` import { NavLink } from ```

Reply:
Prettier is removing these brackets on pushing changes. I can set it to add them but this would apply to all object literals (?)

Reply:
Don't mess with prettier, ignore me!